### PR TITLE
fix(RulesTable): RHICOMPL-2042 custom dedicatedAction Edit Rules

### PR DESCRIPTION
**Pull request by vkrizan**
Fixes showing "Edit Rules" button on Policy details page, Rules tab.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [X] Input Validation
- [X] Output Encoding
- [X] Authentication and Password Management
- [X] Session Management
- [X] Access Control
- [X] Cryptographic Practices
- [X] Error Handling and Logging
- [X] Data Protection
- [X] Communication Security
- [X] System Configuration
- [X] Database Security
- [X] File Management
- [X] Memory Management
- [X] General Coding Practices

### CODE REVIEW

**Review by bastilian — APPROVED** [review]
Awesome! Thank you @vkrizan for fixing this!

**Comment by vkrizan** [comment]
@bastilian PTAL

File: src/PresentationalComponents/RulesTable/RulesTable.js [hunk 1]
```diff
@@ -54,22 +52,12 @@ const RulesTable = ({
...
             identifier: (item) => (item.refId),
-            selectable: !!handleSelect || remediationsEnabled,
+            selectable: !!handleSelect,
             onSelect: handleSelect,
             preselected: selectedRefIds,
             detailsComponent: RuleDetailsRow,
             emptyRows: emptyRows(columns),
-            selectedFilter,
-            dedicatedAction: ({ selected: selectedRules }) => ( // eslint-disable-line
-                remediationsEnabled && <ComplianceRemediationButton
```
Comment (bastilian) [inline src/PresentationalComponents/RulesTable/RulesTable.js:64]: We will need that. Mind trying to apply https://gist.github.com/bastilian/965d53297b3549f59569ab9f391589e1

Comment (bastilian) [reply src/PresentationalComponents/RulesTable/RulesTable.js:64]: @vkrizan Never mind @karelhala will apply the gist, but we will still need to keep the remediations button.

Comment (vkrizan) [reply src/PresentationalComponents/RulesTable/RulesTable.js:64]: @bastilian Just pushed the fix for the dedicated action via #1322.

**Pull request merged**